spec/swagger.yaml

```
@@ -46,18 +46,20 @@ info:
 
     ### Максимальное время обработки запроса
 
-    При любом обращении к API в заголовке соответствующего запроса можно передать параметр отсечки по времени,
-    определяющий максимальное время ожидания завершения операции по запросу:
+    При любом обращении к API в заголовке `X-Request-Deadline` соответствующего запроса можно передать параметр отсечки по времени, определяющий максимальное время ожидания завершения операции по запросу:
 
     ```
      X-Request-Deadline: 10s
     ```
 
     По истечении указанного времени [платформа](https://developer.rbk.money/docs/payments/overview/) прекращает обработку запроса.
 
-      | Формат  | Единицы измерения | Рекомендуемые значения  | Пример 
-      | --         | --          | --         | --         
-      | Формат из раздела "Формат дат" или относительные величины    | `ms` — миллисекунды; `s` — секунды; `m` — минуты    | `3s` < `Request-Deadline` < `1m`    | `150000ms`, `540s`, `3.5m`    | 
+    `X-Request-Deadline` может:
+    
+    * задаваться в формате `date-time` (см. [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339)) или в относительных величинах;
+    * измеряться в миллисекундах (`150000ms`), секундах (`540s`), минутах (`3.5m`).
```

Тогда уж быть может
```suggestion
    * задаваться в формате `date-time` (см. [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339));
    * задаваться в относительных величинах, в миллисекундах (`150000ms`), секундах (`540s`) или минутах (`3.5m`).
```
?